src/renderer/src/components/AccountRoleSelector.tsx

```
@@ -0,0 +1,87 @@
+          control={control}
+          name="accountId"
+          rules={{ required: true }}
+          defaultValue={selectedAccountId}
+          render={({ field }) => (
+            <ComboBox
+              placeHolder="Select Account"
+              selectedOption={selectedAccountId}
+              options={accountOptions}
+              onChange={(option) => {
+                const account = accounts.find((a) => a.accountId === option.value);
+                if (account && account.roles?.[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)